### DISCUSSION TITLE
Show overlay above previous modal when a new modal is opened

### DISCUSSION THREAD

**Discussion by kalon-robson** [faceless-ui/modal#50] [General]
Hi,

I am uncertain if this is already a feature or if I've implemented it incorrectly.

When I open a modal, I can see the overlay, which is the parent for all modals.

When I then open a modal within a modal, it would be visually handy if an overlay was covering the parent modal.

Would you be interested in a proposed PR for this?

Kind Regards,
Kalon Robson

**Comment by jacobsfletch** [comment]
Hey @kalon-robson! Have you seen the drawer example yet? It demonstrates how to apply background overlay to each modal individually, distinct from the top-level modal overlay. Check it out: https://facelessui.com/docs/modal/examples/drawer

**Reply by kalon-robson** [reply]
This worked, thank you, @jacobsfletch
